scim/resource_service_principal_test.go

```diff
@@ -376,9 +376,12 @@ func TestResourceServicePrincipalDelete(t *testing.T) {
 			},
 		},
 		Resource: ResourceServicePrincipal(),
-		HCL:      `display_name = "Squanchy"`,
 		Delete:   true,
 		ID:       "abc",
+		HCL: `
+			display_name = "Squanchy"
+			disable_as_user_deletion = false
```

Comment:
can we make sure not to change the tests?..

Reply:
Done